src/NuGet.Core/NuGet.Protocol/Plugins/PluginFactory.cs

```diff
@@ -165,7 +165,8 @@ public void Dispose()
 #else
             var startInfo = new ProcessStartInfo
             {
-                FileName = Environment.GetEnvironmentVariable("DOTNET_HOST_PATH"),
+                FileName = Environment.GetEnvironmentVariable("DOTNET_HOST_PATH")
+                            ?? (RuntimeEnvironmentHelper.IsWindows ? "dotnet.exe" : "dotnet"),
```

Comment:
Can format this a bit better, put it on a new line to improve readability.

Reply:
Sorry! Would the following be ok? (Applied "Braces may be omitted only if the body of every block associated with an if/else if/.../else compound statement is placed on a single line.")
```csharp
                FileName = Environment.GetEnvironmentVariable("DOTNET_HOST_PATH")
                                    ?? (RuntimeEnvironmentHelper.IsWindows 
                                           ? "dotnet.exe" 
                                           : "dotnet")
```

Reply:
Yeah that's fine, I'd keep the operators on the previous line, but that's just my personal preference :D

Reply:
While we are at, can you please use the PR template? 

We have automation that depends on it.

Reply:
You're the boss ;)

```csharp
FileName = Environment.GetEnvironmentVariable("DOTNET_HOST_PATH") ??
                (RuntimeEnvironmentHelper.IsWindows ? 
                     "dotnet.exe" :   
                     "dotnet")
```
Better?

Reply:
Yes of course for the PR template. Sorry again.

Reply:
Looks great, thanks!
I will kick off the CI. 

Please update the PR template.